crates/uv/tests/run.rs

```diff
@@ -1950,3 +1950,44 @@ fn run_invalid_project_table() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+/// Check warning message for https://github.com/astral-sh/uv/issues/6998.
+fn run_script_without_build_system() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let pyproject_toml = context.temp_dir.child("pyproject.toml");
+    pyproject_toml.write_str(indoc! { r#"
+        [project]
+        name = "foo"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = []
+
+        [project.scripts]
+        entry = "foo:custom_entry"
+        "#
+    })?;
```

Comment:
How does this interact with `package = true` and `package = false`? If the user sets `package = true` we will install the scripts. Similarly, if they set `package = false` we won't install the scripts even if there's a build system declared.